### PR TITLE
Fix: IE8 card styling

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy 

Might be worthwhile having a bespoke file for IE8 styling overwrites rather than resorting to these sorts of selectors, do you think?

### CODE REVIEW

**Comment by ironsidevsquincy** [comment]
What are you trying to achieve? You only want to show these styles for `enhanced` browsers? Why's that?

**Comment by andygout** [comment]
Core browsers don't include lazy-loaded images, meaning we can't assume there will be an image available on which to apply this effect. We could be clever enough to write the styling to apply it when in core to images that are not lazy-loaded, but given this currently amounts to one (the first Life & Arts article), seems excessive, especially given we are talking older browsers.

### Retaining styling (which absolutely positions content)
screen shot 2016-08-01 at 15 41 08

### Removing styling
screen shot 2016-08-01 at 15 34 57

**Comment by ironsidevsquincy** [comment]
So why not use `.enhanced`?

File: components/_card.scss [hunk 1]
```diff
@@ -157,15 +157,17 @@ $rule-size: 1px;
 	}
 
 	&.card__content--image-overlay {
```
Comment (ironsidevsquincy) [inline components/_card.scss]: e.g. `.enhanced &.card__content--image-overlay { ...`

**Comment by andygout** [comment]
Of course!

File: components/_card.scss [hunk 2]
```diff
@@ -165,6 +165,7 @@ $rule-size: 1px;
 			width: auto;
 			padding: 30px 30px 100px;
 			background: linear-gradient(rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));
+			filter: progid:DXImageTransform.Microsoft.gradient(GradientType=0, startColorstr='#000000', endColor=0);
```
Comment (ironsidevsquincy) [inline components/_card.scss]: Should this be part of this PR?

Comment (andygout) [inline components/_card.scss]: Whoops - no (have removed it just now). Am currently looking into an alternative solution for that.

**Comment by ironsidevsquincy** [comment]
👍

**Pull request merged**